lxd/db/cluster/projects.go

```diff
@@ -199,7 +199,7 @@ DELETE FROM projects_config WHERE projects_config.project_id = ?
 	return nil
 }
 
-// InitProjectWithoutImages updates populates the images_profiles table with
+// InitProjectWithoutImages populates the images_profiles table with
```

Comment:
This change should be a separate commit really.

Reply:
Got it, I'll split it.